shot/src/main/scala/com/karumi/shot/ShotPlugin.scala

```diff
@@ -61,28 +58,76 @@ class ShotPlugin extends Plugin[Project] {
   }
 
   private def addTasks(project: Project): Unit = {
+    val appExtension =
+      project.getExtensions.getByType[AppExtension](classOf[AppExtension])
+    val baseTask = project.getTasks.create(
+      Config.defaultTaskName,
+      classOf[ExecuteScreenshotTestsForEveryFlavor])
+    appExtension.getApplicationVariants.all { variant =>
+      val flavor = variant.getMergedFlavor
+      val completeAppId = flavor.getApplicationId + Option(
```

Comment:
It's great we don't have to pass the app id anymore 👏

Reply:
Yeap, I found this method looking at the android gradle plugin code and I decided to remove the config param 😃